z/z-schema/z-schema_rhel_8.4.sh

```diff
@@ -0,0 +1,35 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : z-schema
+# Version       : v4.2.2, v4.2.3 and v5.0.0
+# Source repo   : https://github.com/koolhazz/paping.git
+# Tested on     : RHEL 8.4
+# Script License: Apache License Version 2.0
+# Maintainer    : sachin.kakatkar@ibm.com
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#Run the script:./z-schema_rhel_8.4.sh v5.0.0 
+PACKAGE_NAME=z-schema
+PACKAGE_VERSION=$1
+PACKAGE_URL=https://github.com/zaggino/z-schema.git
+if [ -z "$1" ]
+  then
+    PACKAGE_VERSION=v4.2.2
+fi
+dnf install git wget fontconfig bzip2 npm -y
```

Comment:
this will install node 10 by default while we should be building/testing with node 12.x, please use "dnf module enable nodejs:12" , "dnf install nodejs:12" or nvm to install it.

Reply:
Okay added